.github/wiki/Project-Configurations.md

```diff
@@ -0,0 +1,136 @@
+## Project Configurations
+
+This document presents in detail the set of configurations used in the iOS projects developed at Nimble. Included in this document:
+
+- Terminologies and topics related to project configurations.
+- Basic targets, schemes, build settings options, etc., for building a project.
+
+### **Targets**
+
+A target specifies a product to build, such as an iOS, watchOS, or macOS app.
+
+When creating a project from a template, a default target will be added automatically for the main application
+
+There are 3 default targets in a project:
+
+- {ProjectName}
+- {ProjectName}Tests
+- {ProjectName}UITests
+
+### Schemes
+
+A scheme is a collection of settings that specifies the targets to build for a project, the build configuration to use, and the executable environment to use when the product is launched.
+
+The 2 main schemes in a project:
+
+- {ProjectName}
+- {ProjectName} Staging
+
+### Build Configurations in Schemes
+
+It is possible to build a scheme with different Build Configurations. Initially, there are 2 basic configurations for a project generated automatically by Xcode, which are:
+
+- Debug
+- Release
+
+However, these basic configurations are not enough for our development process. Since our team always wants to ensure the application is working in a designated manner, additional levels of testing are required before the application is ready to be used.
+
+The recommended set of configurations to have is:
+
+- Debug Staging
+- Release Staging
+- Debug Production
+- Release Production
+
```

Comment:
I think it'll be nice to have a break between the terminology and flag option sections.
 ```suggestion
## Predefined Build Options
```

Reply:
@blyscuit Could you tell me more about the details? Do you mean that we will add a new section `Predefined Build Configurations` and explains the terminology of each build configuration?

Reply:
Fixed in https://github.com/nimblehq/ios-templates/pull/268/commits/4ac43d42bd69de88234f85aee75cd8736fdfc56f

Reply:
I think it might improve the document if we add a section break between the top (Project terminology) and bottom (Build configuration that we predefined for the user of the template) to add some sense that we are shifting the content to another topic. What do you think?

Reply:
@blyscuit Good point! Let's migrate the document from Notion first. For content updates, I will create a new issue for this one. What do you think?

Reply:
Sounds good.

Reply:
@blyscuit Created an issue: https://github.com/nimblehq/ios-templates/issues/279